.github/workflows/simple-build.yml

```diff
@@ -23,4 +23,4 @@ jobs:
           java-version: '17'
           distribution: 'temurin'
       - name: Build with Maven
-        run: mvn -version ; mvn clean install -U -B -DskipTests
+        run:  mvn clean install -U -B -DskipTests ; mvn test -pl deployments/intersmash-deployments-provider,tools/intersmash-tools-core,tools/intersmash-tools-provisioners -am
```

Comment:
Do we really not want to see mvn version?

Reply:
Fair enough, fixing.